specs/state-proof.md

```diff
@@ -65,11 +67,13 @@ to not be in the table.
 - 3.3. Stack pointer increases 0 or 1 only
 - 3.4. `initial_value` is 0
 - 3.5. `state root` is the same
+- 3.6. `value_prev` equals `initial_value`
 
 ### Storage
 - 4.0. `field_tag` is 0
 - 4.1. MPT lookup for last access to (address, storage_key)
```

Comment:
There is the constraint "mpt_proof_type is field_tag or NonExistingStorageProof" in the code that is before 4.1.
I'd suggest to either gathering the two under 4.1 and perhaps writing a bit more to explain the constrain, or to add another 4.x for it here